.storybook/preview-head.html

```diff
@@ -25,6 +25,7 @@
 
   #spark-doc-container{
```

Comment:
```suggestion
  #spark-doc-container {
```

Reply:
🙈 Just merged (will fix it in another PR 👍 )